KakaoImageSearch/Api/KakaoImageSearchApi.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  KakaoApi.swift
+//  KakaoImageSearch
+//
+//  Created by 강현석 on 2021/02/01.
+//
+
+import Alamofire
+
+class KakaoImageSearchApi {
+    private let BASE_URL = "https://dapi.kakao.com"
+    private let DOMAIN = "/v2/search/image"
+    
+    func searchImage(
+        keyword: String,
+        page: Int = 1,
+        size: Int = 10
+    ) -> Alamofire.DataRequest {
+        let apiKey = Bundle.main.infoDictionary!["KAKAO_API_KEY"] as! String
+        return AF.request(
+            BASE_URL + DOMAIN,
+            parameters: ["query" : keyword, "page" : page, "size" : size],
```

Comment:
P5: `[Key: Value]` 형식으로 키값과 콜론은 붙이고, Value는 띄워주는게 더 자연스럽습니다.
```suggestion
            parameters: ["query": keyword, "page": page, "size": size],
```

Reply:
P5: 쿼리가 한글이거나 공백이 있을 때 처리가 되어있는지는 PR에 안나와있는것 같은데, 아마 에러가 났던 것 같아요.
처리되어있다면 무시하셔도 좋습니다.

Reply:
안드로이드에서는 인코딩을 라이브러리단에서 해줘서 모르고 있었습니다 😭 
수정했습니다 [df008f0]